src/sentry/static/sentry/app/views/projectInstall/platform.tsx

```diff
@@ -87,35 +98,36 @@ class ProjectInstallPlatform extends React.Component {
     const issueStreamLink = `/organizations/${orgId}/issues/`;
     const performanceOverviewLink = `/organizations/${orgId}/performance/`;
     const gettingStartedLink = `/organizations/${orgId}/projects/${projectId}/getting-started/`;
+    const platformLink = platform.link ?? undefined;
 
     return (
-      <Panel>
-        <PanelHeader hasButtons>
-          {t('Configure %(platform)s', {platform: platform.name})}
-          <Actions>
+      <React.Fragment>
+        <StyledPageHeader>
+          <h2>{t('Configure %(platform)s', {platform: platform.name})}</h2>
```

Comment:
What's the difference between this and `tct`?

Reply:
tct() can format nodes as well.